src/ontology/cl-edit.owl

```diff
@@ -27855,6 +27867,29 @@ SubClassOf(obo:CL_4023050 obo:CL_4023040)
 SubClassOf(obo:CL_4023050 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0001384))
 SubClassOf(obo:CL_4023050 ObjectSomeValuesFrom(obo:RO_0002162 obo:NCBITaxon_10090))
 
+
+
+# Class: obo:CL_4023052 (Betz upper motor neuron)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.1101/2020.03.31.016972") obo:IAO_0000115 obo:CL_4023052 "a Betz cell that synapses with lower motor neurons directly.")
+AnnotationAssertion(obo:RO_0002175 obo:CL_4023052 obo:NCBITaxon_9606)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> obo:CL_4023052 <http://orcid.org/0000-0001-7258-9596>)
+AnnotationAssertion(rdfs:label obo:CL_4023052 "Betz upper motor neuron")
+SubClassOf(obo:CL_4023052 obo:CL_0008049)
+SubClassOf(obo:CL_4023052 ObjectSomeValuesFrom(obo:RO_0002120 obo:CL_0008039))
+
+# Class: obo:CL_4023053 (spinal interneuron syanpsing Betz cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.1101/2020.03.31.016972") obo:IAO_0000115 obo:CL_4023053 "a Betz cell that syanpses with spinal interneurons")
```

Comment:
Worth chasing up some of the refs in DOI:10.1101/2020.03.31.016972 that directly investigate synapsing to see if you could use one of these as an a ref (DOI:10.1101/2020.03.31.016972 doesn't have any connectivity data itself).

Reply:
These should be caught by QC checks + please let me know if not: First letter sentence not capitalised; No full stop at end.